.gitignore

```diff
@@ -143,6 +143,7 @@ settings.json
 **/datasets/news_classification/partitioned_datasets/**
 **/datasets/mnist_data/**
 **/datasets/MNIST/**
+**/datasets/agnews_data/**
```

Comment:
Just formally ignoring the agnews dataset used for the dynamic layer exchanger (if you download it as `agnews_data`)